src/audio/kpb.c

```diff
@@ -2060,6 +2068,50 @@ static inline void kpb_change_state(struct comp_data *kpb,
 	kpb->state_log = (kpb->state_log << 4) | state;
 }
 
+static int kpb_set_micselect(struct comp_dev *dev, const void *data,
+			     int max_data_size)
+{
+	comp_info(dev, "kpb_set_micselect");
```

Comment:
Style note: in SOF code we variables declared before any code, so comp_info() should be after the variable declarations. And this probably should be "comp_dbg()" as this just indicates the function is entered and provides no other information.